pubspec.yaml

```diff
@@ -3,9 +3,8 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.16+5.53.0
+version: 0.5.17+5.54.0
 description: A Dart wrapper around the CodeMirror text editor.
-author: Devon Carew <devoncarew@google.com>
```

Comment:
Sorry @devoncarew, but pub publish keeps hassling me about `author` being deprecated =(

Reply:
yeah, no worries - this field is deprecated